src/stylesheets/elements/_headings.scss

```diff
@@ -68,39 +68,25 @@ h5, h6, .epsilon {
 }
 
 /*--- Heading Nested Elements ---*/
-h1 a {
-  vertical-align: top;
-  @include delta;
-  line-height: 1.2; // 24px
-
-  @include media($medium) {
-    vertical-align: top;
-    line-height: 1.091; // 24px
-  }
-}
-h1 span {
-  display: block;
-  &:last-child {
-    @include fontsize(16);
-    font-weight: 400;
-    line-height: 1.125; // 18px
 
-    @include media($medium) {
-      @include fontsize(18);
-      display: block;
+h1 {
+  @include last-child;
 
-      &:nth-of-type(2) {
-        @include fontsize(18);
-        line-height: 1.334; // 24px
-      }
-    }
+  span {
+    display: block;
   }
-}
-h1 span + span {
-  margin-top: $base-spacing-unit-small;
-
-  .hero & {
-    margin-top: $base-spacing-unit;
+  span.context {
+    @include gamma;
+    @include typography--light;
+    margin: $base-spacing-unit-tiny 0;
+  }
+  > small {
+    top: 0;
+    @include delta;
+    position: absolute;
+  }
+  > small + span {
```

Comment:
For the Roles pages we are developing, we are going to need the span to be an anchor instead. Can we also have this styled for anchor tags as well? Or should we just add the anchor tags inside the span?